index.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="desktop-style.css" />
+    <link rel="stylesheet" href="mobile-style.css" />
+    <title>Capstone</title>
+  </head>
+  <body>
+    <header class="black-menu hide">
+      <ul class="black-menu hide">
+        <li>
+          <a href="#"
+            ><img
+              class="hide link"
+              src="Images/facebook.png"
+              alt="facebook-logo"
+              height="20"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img
+              class="hide link"
+              src="Images/twitter.png"
+              alt="twitter-logo"
+              height="20"
+          /></a>
+        </li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </header>
+
+    
+    <div class="menu">
+      <a href="#" class="disapeer" id="hamburger-menu"
+        ><img
+          class="disapeer"
+          src="Images/menu-mobile.png"
+          alt="hamburger-menu"
+          width="60"
+      /></a>
+      <a class="link hide" href="index.html"
+        ><img
+          class="hide right"
+          src="Images/logo-villa.png"
+          alt="logo"
+          width="135"
+      /></a>
+      <ul class="menu-section">
+        <li>
+          <span class="material-icons close disapeer">close</span>
+        </li>
+        <li><a class="link" href="about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li class="cc"><a href="#">CC Campaign</a></li>
+      </ul>
+    </div>
```

Comment:
The top dark navbar and the bottom white navbar of your desktop version do not match the provided [template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly note we actually want our design to as much as possible _mimic_ the design of the provided template design. Kindly ensure to improve upon these aspects:

- [ ] The content of the _dark navbar_ is **vertically centered** and **left-aligned** with the contents of the white navbar.
- [ ] The **typography** (font family, font-weight and font size, and the like ) **should match** the design of the provided template.
- [ ] The contents on both navbars are **centered** and that that the _padding_ of the extreme left and rights sides of the nav contents are equal
- [ ] The white navbar has a **border-bottom**, as can be seen in the original design below.
- [ ]  Contents of the _dark navbar_ is aligned to the menu item with a thick red border on the right

**Your header navigation sections:**
![image](https://user-images.githubusercontent.com/12745474/133909792-5d2bebd1-04a2-4561-a0d7-046eeceb07a5.png)


<br>

**The original header navigation sections:**
![image](https://user-images.githubusercontent.com/12745474/132956330-393c697b-60c8-4a14-a9a8-0f7397ac04c9.png)